.roo/prompts/pr-review-fixes.md

```
@@ -0,0 +1,7 @@
+I have this PR in my repo https://github.com/WiegerWolf/llm-cli/pull/###
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)